src/react/components/pages/editorPage/tableView.scss

```diff
@@ -1,25 +1,29 @@
 @import "../../../../assets/sass/theme.scss";
 @import "../../common/common.scss";
 
-.viewed-table {
-    border: 1.2px solid rgba(255,255,255, 0.9);
-    td {
-        border: 0.6px solid rgba(255,255,255, 0.7);
-        min-width: 80px;
-        padding: 8px 16px 8px 8px;
-    }
-    td:empty::after {
-        content: "\00a0";
+.table-view-container {
+    direction: rtl;
```

Comment:
why make scroll bar on left side? Typically they're on the right side, right?

Reply:
So it's not too close to table and close btn. I think, this way it looks more balanced.

Reply:
It can be on another side it's really does not matter much to me.

Reply:
I think the balanced style looks good, but maybe should still put it on the right, more follows the common sense.

Reply:
Adding margin to the table adds some separation between the table border and the scrollbar:
![image](https://user-images.githubusercontent.com/60453211/82710723-641dd600-9c38-11ea-9faf-38e2c59f84eb.png)
maybe we should do that instead

Reply:
yeah, I'm moving it back.

Reply:
<img width="1354" alt="Screen Shot 2020-05-22 at 2 49 27 PM" src="https://user-images.githubusercontent.com/64093224/82711873-78170700-9c3b-11ea-9224-b271d1a07a7f.png">